oxo.py

```diff
@@ -1,26 +1,105 @@
 class OxoBoard:
     def __init__(self):
+    # def __init__(self, size)   <-- 1 of 3 to making the code take an size input from play.py
         """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+        self.size = 3
+        # self.size = size    <-- 2 of 3 for variable size input
+        self.board = [[0 for x in range(self.size)] for y in range(self.size)]
+        '''
+         x = 1,2,3
+           [[a,b,c], y = 0
+            [d,e,f], y = 1
+            [g,h,i]] y = 2      i think
+            
+            it'd be pretty easy to implement bigger boards for this, need:
+            -> a screen that asks players how big they want the board
+            -> this __init__ to have an extra parameter for board size
+                -> this parameter also determines grid_width/height in play.py,
+                 the screen size would stay the same, but the boxes are smaller
+            -> the new parameter would be assigned to self.size
+            -> searches for non-diagonal wins would have to be updated, as they
+                are currently hard coded because it was nice and easy :(
+                
+            pep-8 is lame so I didn't stick too strictly to it,
+            comments are sparse because doc-strings exist :)
+            
+            UPDATE:
+            I have changed the play.py variable names a lot, because it annoyed
+            me :) but this allowed me to create my OXO game with the potential
+            to have any sized grid and the appropriate checks for them. All
+            grids will be square because rectangular grids make me the big 
+            angery >:( to change the size of the grid, change the variable
+            ' grid_size '  in play.py to any number ( not 1 or 2 they don't 
+            make for a fun game )
+        '''
 
     def get_square(self, x, y):
         """ Return 0, 1 or 2 depending on the contents of the specified square. """
-        raise NotImplementedError("TODO: implement get_square")
+        return self.board[y][x]
```

Comment:
Why did you decide to have the order be `[y][x]` instead of `[x][y]`? Both obviously work as long as you're consistent, but the latter seems more natural.